package.json

```diff
@@ -27,15 +27,18 @@
   },
   "dependencies": {
     "@azure/msal-node": "^1.1.0",
+    "@types/ioredis": "^4.26.4",
     "axios": "^0.21.1",
     "cookie": "^0.4.1",
     "google-libphonenumber": "^3.2.20",
     "govuk-frontend": "^3.11.0",
+    "ioredis": "^4.27.3",
```

Comment:
The `redis` client package is also available on NPM, and has more weekly downloads.  Both are _very_ widely used, however, and I went with `ioredis` because that's the one used in the [official Next example](https://github.com/vercel/next.js/blob/canary/examples/with-redis/package.json).